azure-pipelines/build-release.yml

```diff
@@ -46,12 +47,13 @@ stages:
         versionSpec: '$(python.version)'
       displayName: 'Use Python $(python.version)'
 
-    # Note the code below generates a pip requirements file from the pipenv development requirements and then installs via
+    # Note the below generates a pip requirements file from the pipenv development requirements and then installs via
```

Comment:
The word `code` was also added in a later commit on my change which makes me worry as I don't think this is quite up to date with master. Also the list of commits seems wrong.

Reply:
the list of commits is due to the fact that this branch was started before your changes and was updated with your changes before you squashed and merged them to `master`.